dcp_prototype/backend/browser/requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+chalice==1.12.0
```

Comment:
[Thoughts](https://github.com/HumanCellAtlas/dcp-prototype/pull/119#discussion_r382159094)?

Reply:
I don't want to bleed our services requirement into other services, especially if we are version locking them.

Reply:
Fair, I'd check in with @parthshahva as well